sympy/solvers/ode/tests/test_ode.py

```diff
@@ -3243,7 +3243,7 @@ def test_issue_9446():
     [Eq(f(x), C1 + pi*x - Integral(asin(f(2*x)), x)), Eq(f(x), C1 + Integral(asin(f(2*x)), x))]
 
     assert integrate(-asin(f(2*x)+pi), x) == -Integral(asin(pi + f(2*x)), x)
-    
+
```

Comment:
Can you leave another blank line here? Just because there's a pattern in this file.

Reply:
I think I fixed that